### ISSUE TITLE
BUG - 2 - iOS - Accessibility: User in multi-line text field has difficulty exiting keyboard

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#10933]
## What happened?

When and iOS user is entering text into a multi-line text field, the screen reader does not make exiting the keyboard or navigating the site easy when the keyboard is visible.

## Specs:

- Device: iPhone 15
- OS: Latest
- User Account: Demo
- Accessibility State: Screen reader ON, default settings.

## Steps to Reproduce

1. As the Demo user, access the Health > Messages > Start new message page.
2. Enable the accessiblility screen reader.
3. Navigate to the 'Subject' text field.
4. When the keyboard appears, enter in some text, then press the 'return' button on the keyboard. Observe that the keyboard overlay closes.
5. Scroll down the page and enter the Message text field. Press the 'return' button on the keyboard. Observe:
 - The keyboard remains on the screen and the user's cursor move down to a new line.
 - Depending on how the user focused on the Message text field, the keyboard can interfere with the display of the message field and Send button. 
 - While the keyboard remains on the sceen, the user has to either swipe through the keyboard keys to navigate to a page element, or  find and select another focusable object on the page to dismiss the keyboard.

## Desired behavior

The user should be able to easily exit the Message field after entering it, just like they can with the Subject line text field.

## Acceptance Criteria

When an accessibility user enters the Message text field. They should be able to easily exit the text field without complications.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: **High** Low 
- Frequency: High **Low** 

- 3 - Low
- 2 - **High**
- 1 - Critical

## Screen shot(s) and additional information

https://github.com/user-attachments/assets/d42110b8-fedc-4d3d-a623-a2b7840a5d9c

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by matthew-df** [comment]
@timwright12 @SKMOstudio Given the desired behavior, does this mean we do not want the user to be able to add a newline to the message and instead close the keyboard?

**Comment by timwright12** [comment]
I'll defer to @SKMOstudio there. Curious what other apps do with multi-line text fields
